src/Sql/Sql/ChangeLog.md

```diff
@@ -18,6 +18,8 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+
+* Added `GeoZone` option to `BackupStorageRedundancy` parameter to `New-AzSqlDatabase`, `Set-AzSqlDatabase`, `New-AzSqlDatabaseCopy`, `New-AzSqlDatabaseSecondary`, and `Restore-AzSqlDatabase` to enable RAGZRS create, update, copy, geo secondary and PITR support for hyperscale databases
```

Comment:
change to "to enable create, update, copy, geo secondary and PITR support for GEOZONE hyperscale databases"